### PR TITLE
AMD to es6 import

### PR DESCRIPTION
**Pull request by haishengwu-okta**
## Description:

- Convert AMD to es6 import syntax for unit test files
- Convert AMD to es6 import syntax for src (v1) plus a small fix.

## PR Checklist

- [ ] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Issue:

- OKTA-323812

### CODE REVIEW

File: src/EnrollDuoController.js [hunk 1]
```diff
@@ -11,118 +11,113 @@
...
+    activate: function (signedResponse) {
+      const url = this.get('postAction');
+      const factorId = this.get('factorId');
+      const self = this;
+      const data = {
+        id: factorId,
+        stateToken: this.get('stateToken'),
+        sig_response: signedResponse,
+      };
+      // Note: We should be doing this in OktaAuth! Fix when it's updated.
```
Comment (aarongranick-okta) [inline src/EnrollDuoController.js:51]: Is there a related JIRA for this?

Comment (haishengwu-okta) [reply src/EnrollDuoController.js:51]: I don think so. I wasn't even aware why it says shall be done in OktaAuth.

Comment (aarongranick-okta) [reply src/EnrollDuoController.js:51]: OK, I just have some confusion on what the actual issue is here. Therefore I have no confidence that any "fix" will be put in.

**Comment by haishengwu-okta** [comment]
move to https://github.com/okta/okta-signin-widget/pull/1395

**Pull request closed without merging**